Automattic/pocket-casts-android
# Bookmarks - Add parent container on episode page

## Description

This is a preparatory PR to add the bookmarks fragment to the episode screen later. 

It
- Adds a parent container for the episode fragment with a view pager to contain child fragments
- Extracts toolbar from episode fragment to the parent container
- Adds tab bar to the toolbar

** I did not use a Compose Pager as the `EpisodeFragament` is not converted to a Compose yet and Compose Pager doesn't seem to support fragments properly. `ViewPager` has a `FragmentStateAdapter` that manages fragment states well.

## Testing Instructions
1. Install debug build
2. Open the episode screen
3. ✅  Notice there's a `Details` tab at the top
4. Disable the bookmarks feature in `Settings` -> `Beta features`
5. Re-open the episode screen
6. ✅  Notice that the `Details` tab is not present

#### Regression

Test that close, star, and share toolbar buttons work as before.

## Screenshots or Screencast 

Bookmarks Enabled | Bookmarks Disabled
----|------
![Screenshot_20230731_180146](https://github.com/Automattic/pocket-casts-android/assets/1405144/b8a0b298-582e-443f-84b4-ac48c2708ee4) | ![Screenshot_20230731_180244](https://github.com/Automattic/pocket-casts-android/assets/1405144/53203443-bdc6-494a-b013-274f8559898e)


## Checklist
 
#### I have tested any UI changes...
